packages/mantine/src/components/modal-wizard/ModalWizard.tsx

```diff
@@ -72,25 +72,32 @@ interface ModalWizardProps {
      */
     modalProps?: any;
 
+    /**
+     * Classname for modal
+     */
+    modalClassName?: any;
+
     /**
      * Label for close button
      */
     closeButtonLabel?: string;
 
+    /**
+     * Props for progress bar
+     */
+    progressBarProps?: any;
+
+    /**
+     * Props for action buttons
+     */
+    buttonProps?: any;
```

Comment:
What would be the use-case for specifying the button props?

Reply:
Something that may be the use case of specifying button props are:

Change button size, which can be useful if I want a small modal wizard:
<img width="658" alt="Screen Shot 2022-11-10 at 11 54 18 AM" src="https://user-images.githubusercontent.com/113382897/201157850-42e274bd-6786-4698-bba0-b2a71248c4ed.png">

There are other possible use cases too, like button radius / variant.

Reply:
The Mantine buttons already have a size prop, I don't think it's necessary to go through the modal wizard props? 🤔 

Also we want all the buttons in the admin to look the same, I don't like the idea of opening the door to custom style 👀

Reply:
I removed buttonProps here, I agree that it is not necessary. I also changed the hardcoded button size from `md` to `sm`